kytos/core/id.py

```diff
@@ -0,0 +1,29 @@
+class SwitchID(str):
```

Comment:
It looks like `SwitchID` encapsulation isn't adding much benefit, I think we could get rid of it and keep it simple and evolve as needed, if ever? and on `InterfaceID` you only set the type as `str`?  wdyt?

Reply:
My main reason for adding in the SwitchID type is to allow programmers to be able to use these types as type hints. Saying you expect a raw string, vs saying you expect a SwitchID give a programmer a different set of expectations. Also in the future, the SwitchID class can be extended, while not requiring additional code refactoring (assuming people use it correctly).